core/domain/email_manager.py

```diff
@@ -63,31 +65,43 @@
     'HTML and include an unsubscribe link.)',
     'You can unsubscribe from these emails from the '
     '<a href="https://www.example.com">Preferences</a> page.')
-# NOTE TO DEVELOPERS: post-signup emails will not be sent if this placeholder
-# is left unmodified. If this policy changes, this should be documented in the
-# wiki.
+# NOTE TO DEVELOPERS: the relevant emails will not be sent if any of these
```

Comment:
This looks like I'm supposed to modify these strings directly in this file, but I can also modify them via the admin config so I am confused.